src/test/java/io/pakland/mdas/githubstats/application/AggregateCommitsTest.java

```diff
@@ -30,38 +29,40 @@ public void aggregatingCommits_shouldGiveValidNumCommits() {
 
     @Test
     public void aggregatingCommits_shouldGiveValidLinesAdded() {
-        List<Commit> commits = new ArrayList<>();
-
-        int totalLines = 0;
-
-        for (int i = 0; i < 10; i++) {
-            int numLines = new Random().nextInt(1000);
-            Commit commit = mock(Commit.class);
-            when(commit.getAdditions()).thenReturn(numLines);
-            commits.add(commit);
-            totalLines += numLines;
-        }
-
-        CommitAggregation commitAggregation = CommitAggregation.aggregate(commits);
-        assertEquals(commitAggregation.getLinesAdded(), totalLines);
+        //TODO: Additions should come form PullRequests.
+//        List<Commit> commits = new ArrayList<>();
+//
+//        int totalLines = 0;
+//
+//        for (int i = 0; i < 10; i++) {
+//            int numLines = new Random().nextInt(1000);
+//            Commit commit = mock(Commit.class);
+//            when(commit.getAdditions()).thenReturn(numLines);
+//            commits.add(commit);
+//            totalLines += numLines;
+//        }
+//
+//        CommitAggregation commitAggregation = CommitAggregation.aggregate(commits);
+//        assertEquals(commitAggregation.getLinesAdded(), totalLines);
```

Comment:
If this code is not correct, I would remove it over commenting it. If we later need it, we have git 🤷‍♂️